esp8266-fastled-webserver/Noise.cpp

```diff
@@ -0,0 +1,418 @@
+/*
```

Comment:
SIMPLE FILE REVIEW

This file exists in various active branches.

* defines `noiseXYZ(...)`, which calculates color
  for a given `x`, `y`, and `z`
* defines `drawNoise()`,  which uses `coordsX` and `coordsY`
  for `x` and `y` respectively
* defines `drawPolarNoise()`, which uses `angles` and `radii`
  for `x` and `y` respectively
* defines the many different pre-defined noise types supported,
  each of which is simply a different combination of color palette
  and seed values for those functions.  Each of those was taken
  from one or more of the active branches.